daily_data/show_data.html.erb

```diff
@@ -3,10 +3,10 @@
  <% end %>
  </ul>
 
- <form action="/add/data" method="post" enctype="application/x-www-form-urlencoded">
+ <%= form_tag("/add/data", method: :post) do -%>
```

Comment:
[`POST` is the default](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-form_tag), so you can omit the `:method` argument if you like.